content/pages/privacy-policy.md

```diff
@@ -0,0 +1,93 @@
+
+For privacy policies and conformance to GDPR, a few distinctions will be done between:
+
+1. Mixxx application
+2. Main Mixxx website
+3. Community Mixxx websites
+
+## 1. Mixxx Application
+
+The Mixxx application doesn't use any kind of Mixxx user account, and doesn't collect any user data, when running.
+
+Mixxx does not collect any data, nor any telemetry, when Mixxx is being run.
+However, some limited personal information could be collected or shared in 3 precise cases:
+
+- for metadata retrieval
+- during broadcasting
+- during the crash report process
+- during automatic updates process
+
+### Mixxx Metadata Requests
+
+The Mixxx Application can automatically search online for metadata associated to your media, on 3rd partiy services.
+This metadata mechanism can be disabled at every moment, in Mixxx preferences.
+The information sent does not contain any name, username, email from the user, but these requests will
+contain information about audio files on your computer, including artist, title, album, etc...
+
+The 3rd party that can be contacted by Mixxx, automatically is:
+
+ - MusicBrainz - For this service, please refer their Privacy Policy here: [https://metabrainz.org/privacy](https://metabrainz.org/privacy)
+
+### Broadcasting
+
+If you use the broadcasting feature of the Mixxx application, you need to enter login data of a third party Soundcast/Icecast server.
+Mixxx itself doesn't offer such a service. Only metadata of the played tracks are send to these servers by the Mixxx application.
+
+### Mixxx Crash Reporting
+
+The Mixxx application itself neither collects crash reports nor shares them.
+
+Crash reports are handled by the Stores/Vendors of each platforms.
+Please refer to those platforms for information about the privacy policies.
+Depending on the platform, Mixxx Core Developers have access to these reports, which usually contain:
+
+- Operating System version
+- CPU and GPU
+- Symbolic StackTrace of the process
+- List of DLLs loaded by the process
+
+None of these data is stored on Mixxx servers.
+In addition Mixxx writes a mixxx.log file which contains entries that may expose personal data. In a support case we may ask to share this file in public. This is a manual process during which you may obfuscate sensitive date before sharing.  
+### Mixxx Update Mechanism
+
+The Mixxx application itself doesn't provide an update mechanism.
+
+In case updates are handled by the platform stores, where the user has downloaded Mixxx from,
+please refer to those platforms for information about the privacy policies.
+
+
+## 2. Main Mixxx Website
+
+The main website of Mixxx, aka www.mixxx.org, does not create or use any user account.
+There are no registration forms, or any similar mechanism.
+Therefore, no personal information is stored at any moment, about the user, by Mixxx.
+
+## 3. Community Mixxx Websites
+
+The community websites from Mixxx are related to development and community, and are the following:

```

Comment:
may be nit-picking
Community Mixxx Websites > Mixxx Community Websites
Main Mixxx Website > Mixxx Main Website
```suggestion
## 3. Mixxx Community Websites

The Mixxx community websites are related to development and community, and are the following:

```